gabarit/template_api/api_project/package_name/model/model_base.py

```diff
@@ -102,18 +100,18 @@ def _load_model(self, **kwargs) -> None:
         """
         settings = ModelSettings(**kwargs)
 
-        logger.info(f"Loading the model from {settings.model_path}")
+        Model.LOGGER.info(f"Loading the model from {settings.model_path}")
```

Comment:
Why is this a class attribute instead of an instance attribute (self.LOGGER) ?

Reply:
Because we use it in a static method